rsv-core/lib/python/rsv/CondorG.py

```diff
@@ -85,8 +85,19 @@ def submit(self, metric, attrs=None, timeout=None):
                 schedd_name = metric.host
             submit_file += "grid_resource = condor %s %s\n\n" % (schedd_name, collector_host)
             submit_file += "remote_universe = local\n"
-            if 'X509_USER_PROXY' in os.environ:
-                submit_file += "x509userproxy = %s\n" % os.environ['X509_USER_PROXY']
+        elif ce_type in ('cream'):
+            self.rsv.log("INFO", "Submitting to CREAM gateway")
+            jobmanager = metric.config_get("jobmanager")
+            if not jobmanager:
+                self.rsv.log("CRITICAL", "CondorG->submit: jobmanager not defined in config")
```

Comment:
Merely logging the error is not enough. You have to abort, because you won't have a usable submit file. For now, add `sys.exit(1)` after the `self.rsv.log` call. We should come up with a better way to abort at some point, but for now, this is what we can do.